e2e_tests/tests/experiment/test_core.py

```diff
@@ -244,28 +218,26 @@ def test_end_to_end_adaptive() -> None:
     assert checkpoint.metadata == db_check.metadata
 
 
-@pytest.mark.e2e_cpu
-def test_log_null_bytes() -> None:
-    sess = api_utils.user_session()
-    config_obj = conf.load_config(conf.fixtures_path("no_op/single.yaml"))
-    config_obj["hyperparameters"]["write_null"] = True
-    config_obj["max_restarts"] = 0
-    config_obj["searcher"]["max_length"] = {"batches": 1}
-    experiment_id = exp.run_basic_test_with_temp_config(
-        sess, config_obj, conf.fixtures_path("no_op"), 1
-    )
-
-    trials = exp.experiment_trials(sess, experiment_id)
-    assert len(trials) == 1
-    logs = exp.trial_logs(sess, trials[0].trial.id)
-    assert len(logs) > 0
-
-
 @pytest.mark.e2e_cpu
 def test_graceful_trial_termination() -> None:
     sess = api_utils.user_session()
-    config_obj = conf.load_config(conf.fixtures_path("no_op/grid-graceful-trial-termination.yaml"))
-    exp.run_basic_test_with_temp_config(sess, config_obj, conf.fixtures_path("no_op"), 2)
+    config = {
+        "hyperparameters": {
+            "actions": {
```

Comment:
why isn't this example just passing in `create_experiment(actions=actions)`?

Reply:
because it needs to be a categorical hyperparameter, which noop.create_experiment does not support.  And I don't think it should support it, because I think there are only one or two places in all 120ish noop tests where this was needed.